.github/workflows/tests.yml

```diff
@@ -30,8 +30,8 @@ jobs:
       - name: "Set the GH workspace as a safe git directory"
         run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
       # --- Build libraries to wasm ---
-      - name: "Build libc"
```

Comment:
This now re-builds `libc` from scratch every time (as it checks-out a fresh version of `llvm` that tracks without `build` directory) so re-building in the GHA takes an outrageous amont of time (>1h).